### PR TITLE
manager: change chart resource multi to single

### PR DESCRIPTION
**Pull request by hunchback**
change array of resources (each a helm chart) to contain only a single value.

### CODE REVIEW

**Review by shlomitk1 — APPROVED** [review]

File: manager/apis/app/v1alpha1/blueprint_types.go [hunk 1]
```diff
@@ -103,9 +103,9 @@ type ComponentTemplate struct {
 	// +required
 	Kind string `json:"kind"`
 
-	// Resources contains the location of the helm chart with info detailing how to deploy
+	// Resource contains the location of the helm chart with info detailing how to deploy
 	// +required
-	Resources []string `json:"resources"`
+	Resource string `json:"resource"`
```
Comment (roee88) [inline manager/apis/app/v1alpha1/blueprint_types.go:108]: Shouldn't we name this chart instead of resource?

**Review by roee88 — APPROVED** [review]

File: manager/apis/app/v1alpha1/blueprint_types.go (hunk 1 above)
Comment (hunchback) [reply manager/apis/app/v1alpha1/blueprint_types.go:108]: will change

File: manager/config/manager/kustomization.yaml [hunk 2]
```diff
@@ -7,5 +7,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/the-mesh-for-data/manager
+  newName: kind-registry:5000/m4d-system/manager
```
Comment (shlomitk1) [inline manager/config/manager/kustomization.yaml:10]: @hunchback Why was this line changed?
